lib/rubocop/cop/util.rb

```diff
@@ -152,6 +152,22 @@ def range_with_surrounding_space(range, side = :both, with_newline = true)
         Parser::Source::Range.new(buffer, begin_pos, end_pos)
       end
 
+      def range_by_whole_lines(range, include_final_newline = false)
```

Comment:
I'd make the second arg a keyword argument. Seems that now we can finally add a cop suggesting using keyword args over optional args, as those are much nicer to work with.

I also dislike the name of the method a bit, as I find it somewhat confusing. Why `range_by_whole_lines`?